{
  "active_lock_reason": null,
  "assignee": null,
  "assignees": [],
  "author_association": "NONE",
  "body": "With version 1.4.7 I could use a datastore that is in a cluster as a vsphere volume as ```volumePath: \"[datastore] path/to/file\"```.  When I upgraded to 1.6.0, this same exact path does not work and throws a ```datastore 'datastore' not found``` error.  I'm struggling to find what changed, but am guessing it's difference in how the datastore is being looked up in vcenter.\r\n\r\nHas anyone had luck with this?\r\n",
  "closed_at": "2017-04-28T18:39:00Z",
  "closed_by": {
    "avatar_url": "https://avatars0.githubusercontent.com/u/13653959?v=4",
    "events_url": "https://api.github.com/users/k8s-github-robot/events{/privacy}",
    "followers_url": "https://api.github.com/users/k8s-github-robot/followers",
    "following_url": "https://api.github.com/users/k8s-github-robot/following{/other_user}",
    "gists_url": "https://api.github.com/users/k8s-github-robot/gists{/gist_id}",
    "gravatar_id": "",
    "html_url": "https://github.com/k8s-github-robot",
    "id": 13653959,
    "login": "k8s-github-robot",
    "node_id": "MDQ6VXNlcjEzNjUzOTU5",
    "organizations_url": "https://api.github.com/users/k8s-github-robot/orgs",
    "received_events_url": "https://api.github.com/users/k8s-github-robot/received_events",
    "repos_url": "https://api.github.com/users/k8s-github-robot/repos",
    "site_admin": false,
    "starred_url": "https://api.github.com/users/k8s-github-robot/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/k8s-github-robot/subscriptions",
    "type": "User",
    "url": "https://api.github.com/users/k8s-github-robot"
  },
  "comments": 7,
  "comments_url": "https://api.github.com/repos/kubernetes/kubernetes/issues/44007/comments",
  "created_at": "2017-04-03T23:14:36Z",
  "events_url": "https://api.github.com/repos/kubernetes/kubernetes/issues/44007/events",
  "html_url": "https://github.com/kubernetes/kubernetes/issues/44007",
  "id": 219089938,
  "labels": [],
  "labels_url": "https://api.github.com/repos/kubernetes/kubernetes/issues/44007/labels{/name}",
  "locked": false,
  "milestone": null,
  "node_id": "MDU6SXNzdWUyMTkwODk5Mzg=",
  "number": 44007,
  "performed_via_github_app": null,
  "repository_url": "https://api.github.com/repos/kubernetes/kubernetes",
  "state": "closed",
  "title": "vsphere volume: Datastore cluster",
  "updated_at": "2017-04-28T18:39:00Z",
  "url": "https://api.github.com/repos/kubernetes/kubernetes/issues/44007",
  "user": {
    "avatar_url": "https://avatars2.githubusercontent.com/u/1779323?v=4",
    "events_url": "https://api.github.com/users/moserke/events{/privacy}",
    "followers_url": "https://api.github.com/users/moserke/followers",
    "following_url": "https://api.github.com/users/moserke/following{/other_user}",
    "gists_url": "https://api.github.com/users/moserke/gists{/gist_id}",
    "gravatar_id": "",
    "html_url": "https://github.com/moserke",
    "id": 1779323,
    "login": "moserke",
    "node_id": "MDQ6VXNlcjE3NzkzMjM=",
    "organizations_url": "https://api.github.com/users/moserke/orgs",
    "received_events_url": "https://api.github.com/users/moserke/received_events",
    "repos_url": "https://api.github.com/users/moserke/repos",
    "site_admin": false,
    "starred_url": "https://api.github.com/users/moserke/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/moserke/subscriptions",
    "type": "User",
    "url": "https://api.github.com/users/moserke"
  }
}